activitysim/examples/prototype_mtc_extended/configs/disaggregate_accessibility.yaml

```diff
@@ -0,0 +1,159 @@
+# Sampling size. 0 = no limit
+# can be whole integer value or a sample rate (percent of zones)
+# zero or missing defaults to full sample!
+DESTINATION_SAMPLE_SIZE: 0.5
+ORIGIN_SAMPLE_SIZE: 5000
+
+# select origin zones weighted by population (or another landuse variable)
+ORIGIN_WEIGHTING_COLUMN: TOTPOP
+
+# Specify the tables to be created and  their variables.
+# Single values are constants
+# Lists are varying and will generate the cartesian product (i.e., all possible non-repeating combinations)
+
+# Example result for households below:
+#hhid   veh hinccat1    hinc    hworkers    persons hht  bldgsz
+# 1     0   1           14000   1           2       1    2
+# 2     1   1           14000   1           2       1    2
+# 3     2   1           14000   1           2       1    2
+# 4     0   2           67000   1           2       1    2
+# 5     1   2           67000   1           2       1    2
+# 6     2   2           67000   1           2       1    2
+# ....
+
+#Note: parameters in ALL CAPS below are required by the program for synthesis
+CREATE_TABLES:
+    PROTO_HOUSEHOLDS:
+        index_col: proto_household_id
+        zone_col: home_zone_id
+        rename_columns:
+            zone_id: home_zone_id
+        VARIABLES:
+            hinccat1: [1, 2, 3, 4]          # Income categories
+            hworkers: 1                  # Household workers
+            veh: [0, 1, 2]               # Household vehicles
+            persons: 2                   # Two persons household
+            HHT: 1                       # Married-couple family household
+            bldgsz: 2                    # Building size - Single family detached
+        # Additional columns that are mapped to another (e.g., hhinccat1: 1 = hinc: 14000)
+        mapped_fields:
+            hinccat1: # List new fields mapped to this field
+              income: # Median income within each quartile
+                1: 15000   # 14000  (for 3 level)
+                2: 45000   # 67000  (for 3 level)
+                3: 80000   # 120000 (for 3 level)
+                4: 150000
+            persons:
+              hhsize:
+                1: 1
+                2: 2
+                3: 3
+            hworkers:
+              num_workers:
+                1: 1
+                2: 2
+                3: 3
+                    
+    PROTO_PERSONS:
+        index_col: proto_person_id
+        VARIABLES:
+            pnum: [1, 2]                # Person number
+            military: 4                 # Not military
+            pstudent: 3                 # Not attending school
+            educ: 13                    # Bachelor's
+            grade: 0                    # Not attending
+            timeFactorWork: 1           # mean
+            timeFactorNonWork: 1        # mean
+        mapped_fields:
```

Comment:
I think users would benefit from more detailed documentation on these settings in case they want to change them. E.g. when does it require users to define mappings for all vs some attributes; what is the minimum required attributes to create/map (seems like at least the ones used in the mode choice and location choice uec specs). I could be wrong but it seems there are some underlying rules like no need to define two workers in the proto pop.

Reply:
Thanks. Where can I find the plan document?